benchmark/base.py

```diff
@@ -10,15 +9,15 @@ def _get_elapsed_time(func_name, arg, lib, repeat, **options):
         elapsed.append(e)
         mem_info.append(m)
     elapsed = sum(elapsed) / len(elapsed)
-    mem_info = {'min': min([m['min'] for m in mem_info]),
-                'max': max([m['max'] for m in mem_info]),
-                'avg': sum([m['avg'] for m in mem_info]) / len(mem_info)}
+    mem_info = {"min": min([m["min"] for m in mem_info]),
+                "max": max([m["max"] for m in mem_info]),
+                "avg": sum([m["avg"] for m in mem_info]) / len(mem_info)}
```

Comment:
요건 사실 별거 아닌데, 블랙돌리면 요런거 스타일도 아래처럼 변하지 않나요??

```suggestion
    mem_info = {
        "min": min([m["min"] for m in mem_info]),
        "max": max([m["max"] for m in mem_info]),
        "avg": sum([m["avg"] for m in mem_info]) / len(mem_info)
    }
```